Formula/kcov.rb

```diff
@@ -0,0 +1,22 @@
+class Kcov < Formula
+  desc "Code coverage tester for compiled programs, Python, and shell scripts"
+  homepage "https://simonkagstrom.github.io/kcov/"
+  url "https://github.com/SimonKagstrom/kcov/archive/v36.tar.gz"
+  sha256 "29ccdde3bd44f14e0d7c88d709e1e5ff9b448e735538ae45ee08b73c19a2ea0b"
+  head "https://github.com/SimonKagstrom/kcov.git"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+
+  def install
+    mkdir "build" do
+      system "cmake", "-DSPECIFY_RPATH=ON", *std_cmake_args, ".."
+      system "make"
+      system "make", "install"
+    end
+  end
+
+  test do
+    system "#{bin}/kcov", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (`foo --version` or `foo --help`) are not sufficient, as explained in the [formula cookbook](https://docs.brew.sh/Formula-Cookbook).

In most cases, a good test would involve running a simple test case: run `#{bin}/foo input.txt`.
- Then you can check that the output is as expected (with `assert_equal` or `assert_match` on the output of `shell_output`)
- You can also check that an output file was created, if that is expected: `assert_predicate testpath/"output.txt", :exist?`

Some advice for specific cases:
- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Please also look at discussion in previous `kcov` pull requests

Reply:
Sure I'll add a better test. Don't see anything relevant in the previous pull requests. Was there something in particular?

Reply:
Note new test works locally for me, but not on CI, it seems. Not sure the issue yet.